watertap/unit_models/steam_heater.py

```diff
@@ -0,0 +1,116 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+from pyomo.environ import Var
+
+from idaes.core import (
+    declare_process_block_class,
+)
+from idaes.models.unit_models.heat_exchanger import HeatExchangerData
+from idaes.core.solvers import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+import idaes.logger as idaeslog
+
+
+_log = idaeslog.getLogger(__name__)
+
+
+__author__ = "Elmira Shamlou"
+
```

Comment:
I might add a doc string up here describing the model, its relation to the IDAES version that it is based off of, and describe why this separate implementation is being used instead.

Reply:
Oh, and you can also add a TODO note along the lines of "Consider incorporating as a modification to IDAES' FeedWaterHeater model directly in the IDAES repo."

Reply:
addressed